client/src/locales/en.json

```diff
@@ -152,7 +152,6 @@
   "list.list-item.clone": "Clone item",
   "list.list-item.confirmation": "Do you really want to archive '{name}' item?",
   "list.list-item.author": "Added by: {authorName}",
-  "list.list-item.busy-info": "This item is currently edited by someone.",
```

Comment:
no more text about locked item?

Reply:
no, because currently we are only disabling currently edited field like name, description.